optimization/optimization_pr.md

```diff
@@ -0,0 +1,399 @@
+
+Let's attempt to fit parameters using the less-smoothed output by running another optimization round. Leave the optimization configuration at the default number of simulations and parameter ranges. After the optimization completes, you should see a plot like Figure 12 below, where the RMSE is reduced to 8.49, a very good fit.
+
+<div style="background-color:rgba(0, 0, 0, 0); margin-top:20px; text-align: center; vertical-align: middle; margin-bottom:40px;">
+
+<h3>Figure 12</h3>
+
+<a href="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image18.png">
+<img src="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image18.png" alt="image18" width=80%/>
+</a>
+<p>Fit after optimization using simulation trials with a 15 ms smoothing window
+</p>
+</div>
+
+Note that there were minimal changes to the first and second proximal inputs. For those inputs changes were 75% or less, but for the distal evoked input, AMPA weights changed significantly. The plot reflects this in that the regions around the proximal inputs changed minimally, but the negative trough immediately following the distal input is fit very closely by the optimized parameters. The downward deflection follows the slope of the experimental data closely and nearly reaches its minimum at the same time, around 75 ms.
+
+Through a total of 4 optimizations from a baseline parameter set designed for a different experimental scenario, we have found parameters matching new experimental data.
+
+<div style="background-color:rgba(0, 0, 0, 0); text-align:center; vertical-align: middle; margin-top:20px; margin-bottom:40px">
+
+<table style="border-collapse: collapse; border: none;">
+
+<tr style="border: none;">
+<h3>Figure 13</h3>
+</tr>
+
+<tr style="border: none;">
+
+<td style="border: none;" width="33%">
+<a href="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image19.png"><img src="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image19.png" alt="image19" width=100%/>
+</a>
+<p>  </p>
+</td>
+
+<td style="border: none;" width="33%">
+<a href="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image20.png"><img src="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image20.png" alt="image20" width=100%/>
+</a>
+<p>  </p>
+</td>
+
+<td style="border: none;" width="33%">
+<a href="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image21.png"><img src="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image21.png" alt="image21" width=100%/>
+</a>
+<p> </p>
+</td>
+
+</tr>
+
+</table>
+
+<p>Parameter values and relative changes in the final optimization shown in Figure 12</p>
+
+</div>
+
+## Identifying key parameters
```

Comment:
I like this section since it provides intuition on origins of the waveform features